ouroboros-network/test/Test/Socket.hs

```diff
@@ -198,6 +198,7 @@ prop_socket_send_recv initiatorAddr responderAddr f xs = do
 
     cv <- newEmptyTMVarM
     sv <- newEmptyTMVarM
+    siblingVar <- newTVarM 2
```

Comment:
@karknu could you put the commit explanation here in the test. It will be easier to discover.